### PR TITLE
Tabular result

### PR DESCRIPTION
**Pull request by allenzhong**
Tabular result and re-organized content.
However, since other team changed table column, I don't know how many column should be displayed in results table?
If you guys can add these, please work on this branch directly.

### CODE REVIEW

File: app/views/search/_results.html.erb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            <div class="col-sm-offset-1 col-sm-9 col-md-offset-1 col-md-9">
+              <strong>Source:</strong> <small> <%= evidence.source_title%> </small>
+            </div>
+          </div>
+          <div class="row">
+            <div class="col-sm-offset-1 col-sm-9 col-md-offset-1 col-md-9">
+              <strong>URL:</strong><small> <%= evidence.URL %> </small>
+              <strong>DOI:</strong><small><%= evidence.DOI%></small>
+            </div>
+          </div>
```
Comment (chandleryang76) [inline app/views/search/_results.html.erb:98]: I think if  
Author,Title, Category,Publisher,publish date,URL,DOI, Research Aim display,,
it is enough.

Comment (allenzhong) [reply app/views/search/_results.html.erb:98]: I will add missing columns

Comment (chandleryang76) [reply app/views/search/_results.html.erb:98]: Good~

**Review by JohnnyJ0622 — APPROVED** [review]
It's ok.

**Comment by Isabella930514** [comment]
OK

**Pull request merged**